core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ddl.scala

```diff
@@ -119,6 +119,21 @@ private[crossdata] case class CreateView(viewIdentifier: TableIdentifier, queryP
   }
 }
 
+case class CreateExternalTable(
+                                tableIdent: TableIdentifier,
+                                userSpecifiedSchema: StructType,
+                                provider: String,
+                                options: Map[String, String]) extends LogicalPlan with RunnableCommand {
+
+
+  override def run(sqlContext: SQLContext): Seq[Row] = {
+
+    val resolved = ResolvedDataSource.lookupDataSource(provider).newInstance()
 
+    val tableManipulation = resolved.asInstanceOf[TableManipulation]
```

Comment:
The datasource may not be a TableManipulation, so we should define how to handle it. Options:
 -> Throw an exception 
->  Use sys.error like most spark runnable commands